src/helpers/enrich-dataset.test.ts

```diff
@@ -0,0 +1,585 @@
+import { enrichDataset } from "./enrich-dataset";
+import * as _ from 'lodash';
+
+jest.mock("@esri/hub-search");
+
+describe('enrichDataset function', () => {
+    const hubsite = {
+        siteUrl: 'arcgis.com',
+        portalUrl: 'portal.arcgis.com',
+        orgBaseUrl: 'qa.arcgis.com',
+        orgTitle: "QA Premium Alpha Hub"
+    }
+
+    it('should return geojson with enriched fields in properties field', async () => {
+        const hubDataset = {
+            owner: 'fpgis.CALFIRE',
+            created: 1570747289000,
+            modified: 1570747379000,
+            tags: ['Uno', 'Dos', 'Tres'],
+            extent: {
+                coordinates: [
+                    [-123.8832, 35.0024],
+                    [-118.3281, 42.0122],
+                ],
+                type: 'envelope',
+            },
+            name: 'DCAT_Test',
+            description: 'Some Description',
+            source: 'Test Source',
+            id: '123a_0',
+            type: 'Feature Layer',
+            url: 'https://services1.arcgis.com/jUJYIo9tSA7EHvfZ/arcgis/rest/services/DCAT_Test/FeatureServer/0',
+            layer: {
+                geometryType: 'esriGeometryPolygon',
+            },
+            server: {
+                spatialReference: {
+                    latestWkid: 3310,
+                    wkid: 3310,
+                },
+            },
+            identifier: 'CALFIRE::DCAT_Test',
+            slug: 'CALFIRE::DCAT_Test'
+        };
+
+
+        const expectedEnrichedProperties = {
+            owner: 'fpgis.CALFIRE',
+            created: 1570747289000,
+            modified: 1570747379000,
+            tags: ['Uno', 'Dos', 'Tres'],
+            extent: { coordinates: [[-123.8832, 35.0024], [-118.3281, 42.0122]], type: 'envelope' },
+            name: 'DCAT_Test',
+            description: 'Some Description',
+            source: 'Test Source',
+            id: '123a_0',
+            type: 'Feature Layer',
+            url: 'https://services1.arcgis.com/jUJYIo9tSA7EHvfZ/arcgis/rest/services/DCAT_Test/FeatureServer/0',
+            layer: { geometryType: 'esriGeometryPolygon' },
+            server: { spatialReference: { latestWkid: 3310, wkid: 3310 } },
+            identifier: 'CALFIRE::DCAT_Test',
+            slug: 'CALFIRE::DCAT_Test',
+            ownerUri: 'qa.arcgis.com/sharing/rest/community/users/fpgis.CALFIRE?f=json',
+            language: '',
+            keyword: ['Uno', 'Dos', 'Tres'],
+            issuedDateTime: '2019-10-10T22:41:29.000Z',
+            orgTitle: 'QA Premium Alpha Hub',
+            provenance: '',
+            hubLandingPage: 'https://arcgis.com/maps/CALFIRE::DCAT_Test',
+            downloadLink: 'https://arcgis.com/datasets/CALFIRE::DCAT_Test',
```

Comment:
`downloadLink` is used in RSS feed [here](https://github.com/ArcGIS/hub-feeds-api/blob/main/src/default-feed-templates/rss/template.js#L17). I understand it can be misleading as it not actually a download link. I don't recall reasoning for the name. I have added to do comment to change it.